### DISCUSSION TITLE
photoshop plugin?

### DISCUSSION THREAD

**Discussion by Delanyredo** [xinntao/Real-ESRGAN#511] [Ideas]
hey 
we would love if u guys made a plugin for photoshop 
since 80% of designers use photoshop to work

**Comment by lightshadown** [comment]
Any idea what language is needed for photoshop plugins? i was thinking the plugin uses google colab since theres not that many people that have extra powerfull machines, even when working with photograpy, unless you do video then it can be process locally

**Reply by Delanyredo** [reply]
photoshop plugin uses js u can create it via photoshop sdk , it would be nice to include the option (online-offline )
online with google colab and offline locally >
